### PR TITLE
CSS-canvas-medical exercise - Ashley Ho

### PR DESCRIPTION
**Pull request by ashleylho**
https://user-images.githubusercontent.com/60860374/187110643-0de26a21-4029-4e86-a840-f567aeb94b03.mov

Screen Shot 2022-08-28 at 7 23 15 PM
Screen Shot 2022-08-28 at 7 23 30 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Ashley! The logos for Canvas are images, not text, they were provided in your starter files for the project. Additionally, the "Contact Us" button in the header is too tall as compared to the example, please revise and submit a new screenshot with your revision.

**Comment by ashleylho** [comment]
Screen Shot 2022-08-29 at 8 44 54 PM
Screen Shot 2022-08-29 at 8 45 09 PM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+h2 {
+  font-size: 28px;
+}
+
+.two-column,
+.two-column-company-product {
+  margin: 0 auto;
+  width: 50%;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:136]: When naming `column` classes, the `column` portion should come before the measurement, so it is clear at a glance that it is meant to represent a `column` layout class. Also, `two` does not make it clear what the size for this class is. I would rename this to `column-half`. Please make sure to do the same on any other `column-` classes you have in your layout.

Additionally, you should not create variants of `column-` classes based on where it is being applied. You should just use one `column-half` class for all situations requiring a 50% width column.

Lastly, `column-` classes should only include a `flex-basis` or `width` property, and nothing else. You should move the `margin` on this class to another class that you can add to the element in addition to the `column-` class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**